src/ceph.in

```diff
@@ -1032,21 +1032,35 @@ def main():
         return 1
 
     if parsed_args.help:
-        hdr('Monitor commands:')
+        target = None
+        if len(childargs) >= 2 and childargs[0] == 'tell':
+            target = childargs[1].split('.')
+            if not validate_target(target):
+                print('target {0} doesn\'t exists, please pass correct target to tell command, such as mon.a/'
```

Comment:
```suggestion
                print('target {0} doesn\'t exist, please pass correct target to tell command, such as mon.a/'
```

Reply:
nit, i'd suggest put
```py
        target = None
        if len(childargs) >= 2 and childargs[0] == 'tell':
            target = childargs[1].split('.')
            if not validate_target(target):
                print('target {0} doesn\'t exist, please pass correct target to tell command, such as mon.a/'
                      'osd.1/mds.a/mgr'.format(childargs[1]), file=sys.stderr)
                return 1
            childargs = childargs[2:]
            hdr('Tell %s commands:' % target[0])
        else:
            target = ('mon', '')
            hdr('Monitor commands:')
```
to avoid two `if`s.